src/test/java/fi/otavanopisto/kuntaapi/test/server/integration/gtfs/RouteTestsIT.java

```diff
@@ -0,0 +1,131 @@
+    getPtvMocker().endMock();
+    deletePtvSettings();
+    deleteGtfsSettings(organizationId);
+  }
+
+  @Test
+  public void testListRoutes() {
+    String organizationId = getOrganizationId(0);
+    
+    Response response = given() 
+      .baseUri(getApiBasePath())
+      .contentType(ContentType.JSON)
+      .get("/organizations/{organizationId}/transportRoutes", organizationId);
+    
+    response
+      .then()
+      .assertThat()
+      .statusCode(200)
+      .body("id.size()", is(3))
+      .body("id[1]", notNullValue())
+      .body("shortName[1]", is("1 A"))
+      .body("longName[1]", is("Tori-Tuppurala-Tori"))
+      .body("agencyId[1]", notNullValue());
+    
+    String agencyId = response.body().jsonPath().getString("agencyId[1]");
+    assertFound(String.format("/organizations/%s/transportAgencies/%s", organizationId, agencyId));
+  }
+  
+  @Test
+  public void testFindRoute() {
+    String organizationId = getOrganizationId(0);
+    Response response = given() 
+      .baseUri(getApiBasePath())
+      .contentType(ContentType.JSON)
+      .get("/organizations/{organizationId}/transportRoutes/{routeId}", organizationId, getOrganizationRouteId(organizationId, 2));
+    
+    response
+      .then()
+      .assertThat()
+      .statusCode(200)
+      .body("id", notNullValue())
+      .body("shortName", is("1 AB"))
+      .body("longName", is("Tori-Tuppurala-Tori"))
+      .body("agencyId", notNullValue());
+    
+    String agencyId = response.body().jsonPath().getString("agencyId");
+    assertFound(String.format("/organizations/%s/transportAgencies/%s", organizationId, agencyId));
+  }
+  
+  @Test
+  public void testOrganizationRoutesNotFound() throws InterruptedException {
+    String organizationId = getOrganizationId(0);
+    String incorrectOrganizationId = getOrganizationId(1);
+    String organizationRouteId = getOrganizationRouteId(organizationId, 0);
+    
+    String[] malformedIds = new String[] {"evil", "*", "/", "1", "-1", "~"};
+    assertFound(String.format("/organizations/%s/transportRoutes/%s", organizationId, organizationRouteId));
+    assertEquals(3, countApiList(String.format("/organizations/%s/transportRoutes", organizationId)));
+    
+    for (String malformedId : malformedIds) {
+      assertNotFound(String.format("/organizations/%s/transportRoutes/%s", organizationId, malformedId));
+    }
+    
+    assertNotFound(String.format("/organizations/%s/transportRoutes/%s", incorrectOrganizationId, organizationRouteId));
+    assertEquals(0, countApiList(String.format("/organizations/%s/transportRoutes", incorrectOrganizationId)));
+  }
+
+  private void createPtvSettings() {
+    insertSystemSetting(PtvConsts.SYSTEM_SETTING_BASEURL, String.format("%s%s", getWireMockBasePath(), BASE_URL));
+    flushCache();
+  }
+  
+  private void deletePtvSettings() {
+    deleteSystemSetting(PtvConsts.SYSTEM_SETTING_BASEURL);
+  }
+  
+  private void createGtfsSettings(String organizationId) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused private "createGtfsSettings" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUnusedPrivateMethod)